### PR TITLE
Logging for TimesIncludeMillisecondPart test and reenable for Linux

### LINKED ISSUE #26349: Test failed:  System.IO.FileSystem.Tests / TimesIncludeMillisecondPart
```
Assert.All() Failure: 9 out of 9 items in the collection did not pass.
[8]: Xunit.Sdk.NotEqualException: Assert.NotEqual() Failure
     Expected: Not 0
     Actual:   0
        at Xunit.Assert.NotEqual[T](T expected, T actual, IEqualityComparer`1 comparer)
        at System.IO.Tests.BaseGetSetTimes`1.<>c__DisplayClass8_0.<TimesIncludeMillisecondPart>b__0(TimeFunction function) in /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs:line 86
        at Xunit.Assert.All[T](IEnumerable`1 collection, Action`1 action)
```

### PR DESCRIPTION
**Pull request by danmosemsft**
For Windows, it has failed sporadically on Windows 7. Add some logging.

For Unix, leave test disabled, but add extra logging to see which distros are affected. 

Updated:
Fixes #26349

### CODE REVIEW

**Comment by danmosemsft** [comment]
Unix issue is https://github.com/dotnet/corefx/issues/26349

**Comment by danmosemsft** [comment]
@dotnet-bot help please

**Comment by danmosemsft** [comment]
@dotnet-bot test Linux x64 Debug Build
@dotnet-bot test OSX x64 Release Build
@dotnet-bot test Alpine.3.6 x64 Release Build

**Comment by danmosemsft** [comment]
Seems to be passing 100% on Linux, just not OSX. I will enable test again...

**Comment by danmosemsft** [comment]
@dotnet-bot test Linux x64 Debug Build
@dotnet-bot test OSX x64 Release Build
@dotnet-bot test Alpine.3.6 x64 Release Build

**Comment by danmosemsft** [comment]
Fixes  #26349

File: src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs [hunk 1]
```diff
@@ -68,26 +69,52 @@ public void CanGetAllTimesAfterCreation()
...
 
+                    // This case should only happen 1/1000 times, unless the OS/Filesystem does
+                    // not support millisecond granularity.
+
+                    // If it's 1/1000, or low grandularity, this may help:
+                    Thread.Sleep(1234);
+
+                    // If it's the OS/Filesystem often returns 0 for the millisecond part, this may
+                    // help prove it:
+                    //Console.WriteLine($"TimesIncludeMillisecondPart got a file time of {time.ToString("o")}");
```
Comment (Anipik) [inline src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs]: nit : space after ```//```

File: src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs [hunk 2]
```diff
@@ -68,26 +69,52 @@ public void CanGetAllTimesAfterCreation()
...
+        // OSX does not currently support millisec granularity: use this test as a canary to flag
+        // if this ever changes so we can enable the actual test
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public void TimesIncludeMillisecondPart_OSX()
+        {
+            T item = GetExistingItem();
+            Assert.All(TimeFunctions(), (function) =>
+            {
+                DateTime time = function.Getter(item);
+
```
Comment (Anipik) [inline src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs]: nit : Extra Line

**Comment by danmosemsft** [comment]
Tizen is https://github.com/dotnet/coreclr/issues/16679 and we historically do not block on Tizen when it is broken.

**Pull request merged**